gslib/commands/cp.py

```diff
@@ -81,194 +81,188 @@
+  In addition, the resulting names depend on whether the destination subdirectory
+  exists. For example, if ``gs://my-bucket/subdir`` exists as a subdirectory,
+  the following command creates the object ``gs://my-bucket/subdir/dir2/a/b/c``:
 
     gsutil cp -r dir1/dir2 gs://my-bucket/subdir
 
-  will create the object gs://my-bucket/subdir/dir2/a/b/c. In contrast, if
-  gs://my-bucket/subdir does not exist, this same gsutil cp command will create
-  the object gs://my-bucket/subdir/a/b/c.
+  In contrast, if ``gs://my-bucket/subdir`` does not exist, this same ``gsutil cp``
+  command creates the object ``gs://my-bucket/subdir/a/b/c``.
 
-  NOTE: If you use the
+  NOTE: The
   `Google Cloud Platform Console <https://console.cloud.google.com>`_
-  to create folders, it does so by creating a "placeholder" object that ends
+  creates folders by creating "placeholder" objects that end
   with a "/" character. gsutil skips these objects when downloading from the
-  cloud to the local file system, because attempting to create a file that
-  ends with a "/" is not allowed on Linux and macOS. Because of this, it is
-  recommended that you not create objects that end with "/" (unless you don't
-  need to be able to download such objects using gsutil).
+  cloud to the local file system, because creating a file that
+  ends with a "/" is not allowed on Linux and macOS. We
+  recommend that you only create objects that end with "/" if you don't
+  intend to download such objects using gsutil.
 """
 
 _SUBDIRECTORIES_TEXT = """
 <B>COPYING TO/FROM SUBDIRECTORIES; DISTRIBUTING TRANSFERS ACROSS MACHINES</B>
   You can use gsutil to copy to and from subdirectories by using a command
-  like:
+  like this:
 
     gsutil cp -r dir gs://my-bucket/data
 
-  This will cause dir and all of its files and nested subdirectories to be
+  This causes ``dir`` and all of its files and nested subdirectories to be
   copied under the specified destination, resulting in objects with names like
-  gs://my-bucket/data/dir/a/b/c. Similarly you can download from bucket
-  subdirectories by using a command like:
+  ``gs://my-bucket/data/dir/a/b/c``. Similarly, you can download from bucket
+  subdirectories using the following command:
 
     gsutil cp -r gs://my-bucket/data dir
 
-  This will cause everything nested under gs://my-bucket/data to be downloaded
-  into dir, resulting in files with names like dir/data/a/b/c.
+  This causes everything nested under ``gs://my-bucket/data`` to be downloaded
+  into ``dir``, resulting in files with names like ``dir/data/a/b/c``.
 
   Copying subdirectories is useful if you want to add data to an existing
   bucket directory structure over time. It's also useful if you want
   to parallelize uploads and downloads across multiple machines (potentially
-  reducing overall transfer time compared with simply running gsutil -m
+  reducing overall transfer time compared with running ``gsutil -m``
```

Comment:
I think the closing `` comes too early, and that cp (on the next line) should be part of the code block